### PR TITLE
Link against libtirpc when build fails due to missing xdr_* symbols.

### PR DESCRIPTION
**Pull request by plinnie**
I had this problem when building for Ubuntu under WSL2. Uncommenting this line fixed it for me.

### CODE REVIEW

**Comment by ralphlange** [comment]
ASYN is not part of EPICS Base. EPICS Base does not use RPC.

Not sure if this is the right place to document this.

**Comment by plinnie** [comment]
Well, I think everyone using an modern Ubuntu (Debian?) variant working through the getting-started guide will run into this I would think. I'm going to install it on Mint this afternoon. If I run into this problem again, I'll let you know.

**Comment by ralphlange** [comment]
Sorry, my bad.
I thought this PR was against the EPICS Base documentation.

**Comment by plinnie** [comment]
> Sorry, my bad. I thought this PR was against the EPICS Base documentation.

Alright, well, feel free to word is otherwise, or relocated it. But I think it would be good to add.

File: getting-started/installation-linux.rst [hunk 1]
```diff
@@ -252,6 +252,16 @@ Now, run
 
     make
 
+If the build fails due to implicit delcaration of ``xdr_*`` functions it is likely async that should build against libtirpc. To do so you can uncomment ``# TIRCP=YES`` in ``configure/CONFIG_SITE`` of asyn, such that it states:
```
Comment (ralphlange) [inline getting-started/installation-linux.rst:255]: typos: delcaration -> declaration, async -> asyn, TIRCP -> TIRPC
missing comma after "To do so"

**Review by ralphlange — CHANGES_REQUESTED** [review]

File: getting-started/installation-linux.rst (hunk 1 above)
Comment (plinnie) [reply getting-started/installation-linux.rst:255]: oops :-) Thanks

Comment (plinnie) [reply getting-started/installation-linux.rst]: I've fixed it.

**Comment by ralphlange** [comment]
Thank you!!

**Pull request merged**